crates/kitsune_p2p/types/src/metrics.rs

```diff
@@ -307,6 +307,7 @@ impl MetricTaskCounter {
 }
 
 #[tokio::test(flavor = "multi_thread")]
+// @neonphog any idea why?
```

Comment:
maybe other tests running in parallel are using up our tokio core execution threads? Would it pass if we set the final sleep to `50` ms?